src/latex-to-omml.ts

```
@@ -0,0 +1,486 @@
+
+    // Functions
+    if (KNOWN_FUNCTIONS.has(cmd.slice(1))) {
+      const arg = this.parseGroup();
+      return '<m:func><m:fName>' + makeStyledRun(cmd.slice(1)) + '</m:fName><m:e>' + arg + '</m:e></m:func>';
+    }
+
+    switch (cmd) {
+      case '\\frac':
+        const num = this.parseGroup();
+        const den = this.parseGroup();
+        return '<m:f><m:num>' + num + '</m:num><m:den>' + den + '</m:den></m:f>';
```

Biome (2.3.14)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)